scenes/game_elements/props/interact_area/components/interact_area.gd

```diff
@@ -1,13 +1,22 @@
 # SPDX-FileCopyrightText: The Threadbare Authors
 # SPDX-License-Identifier: MPL-2.0
+@tool
 class_name InteractArea
 extends Area2D
 
 signal interaction_started(player: Player, from_right: bool)
 signal interaction_ended
 
+const EXAMPLE_INTERACTION_FONT = preload("res://assets/fonts/m6x11plus.ttf")
```

Comment:
Maybe "DEFAULT_INTERACTION_FONT" is better? We have other cases in other scenes named "default...".

Reply:
> Maybe "DEFAULT_INTERACTION_FONT" is better? We have other cases in other scenes named "default...".

That could work too, I kind of wanted to reflect that this label that is shown here is just representative of where the label will be positioned, but the actual interaction label might look different or even have a different font.